src/Chunk.cpp

```diff
@@ -2278,6 +2299,38 @@ bool cChunk::DoWithBeaconAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBeaconCal
 
 
 
+bool cChunk::DoWithBrewingstandAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBrewingstandCallback & a_Callback)
+{
```

Comment:
I agree that it's become too much. However, I don't think that this PR is the right place to fix it. Let's accept it for now and refactor the entire thing later in a separate PR.
Just a note, the Lua API already has these and they're used a lot, so we'll need compatibility bindings, possibly templatized.

worktycho <notifications@github.com> wrote:

>> @@ -2278,6 +2299,38 @@ bool cChunk::DoWithBeaconAt(int a_BlockX, int
>a_BlockY, int a_BlockZ, cBeaconCal
>>  
>>  
>>  
>> +bool cChunk::DoWithBrewingstandAt(int a_BlockX, int a_BlockY, int
>a_BlockZ, cBrewingstandCallback & a_Callback)
>> +{
>
>Right, and this gives me the opportunity to bring up the fact that I
>don't think we should be doing that. It adds to much complexity, and
>redundant code to cChunk/cWorld and cChunkMap.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/cuberite/cuberite/pull/2254/files#r32614669

-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.